Makefile

```diff
@@ -21,6 +21,22 @@ all: build
 
 build: version build-venv
 
+# Similar to the set in build-venv-reqs, but installs to the global python
```

Comment:
Correct, that means we need system packages for python-django, python-djangorestframework, python-dateutil, python-zerorpc, python-gevent, python-pytz and python-setuptools. We've got all those on SLES and openSUSE. I'm not sure if they're all present on Fedora and RHEL (also, their names might be different).